lib/rbeapi/api/users.rb

```diff
@@ -196,11 +196,11 @@ def parse_user_entry(user)
       # @option :opts [String] :sshkey The sshkey value to assign to the user
       #
       # @return [Boolean] returns true if the command completed successfully
-      def create(name, nopassword = false, secret = nil, opts = {})
+      def create(name, opts = {})
```

Comment:
You need to update the method documentation above to match the parameter changes to the method.